current/runtime/fees.md

```diff
@@ -4,118 +4,151 @@ lang: en
 title: Transaction Fees
 ---
 
-When transactions are submitted to a blockchain, they are executed by the nodes in the network. To
-be economically sustainable, nodes charge a fee to execute a transaction. This fee must be covered
-by the sender of the transaction. The cost to execute transactions can vary, so Substrate provides a
-flexible mechanism to characterize the minimum cost to include a transaction in a block.
+When transactions (extrinsics) are submitted to a blockchain, they are executed (dispatched) by the
+nodes in the network. To be economically sustainable, nodes charge a fee to dispatch a transaction.
+This fee must be covered by the sender of the transaction. The cost to dispatch transactions can
+vary, so Substrate provides a flexible mechanism called "weights" to characterize the minimum cost to
+include a transaction in a block.
 
 The fee system is heavily linked to the [weight system](/current/learn-substrate/weight.md). Make 
 sure to understand weights before reading this document.
 
-## Default Fees
+## Inclusion Fee
 
-The default fee system uses the uses _fixed_ weights represented by the following enum:
+A transaction fee consists of three parts:
+
+* `base_fee`: A fixed fee that is applied to every transaction. See
+  [`TransactionBaseFee`](https://substrate.dev/rustdocs/master/pallet_transaction_payment/trait.Trait.html#associatedtype.TransactionBaseFee).
+* `length_fee`: A per-byte fee that is multiplied by the number of bytes in the encoded transaction. See
+  [`TransactionByteFee`](https://substrate.dev/rustdocs/master/pallet_transaction_payment/trait.Trait.html#associatedtype.TransactionByteFee).
+* `weight_fee`: A per-weight-unit fee that is multiplied by the weight of the transaction. The
```

Comment:
You've corrected your own mistake here. A per-weight fee is not multiplied by the weight of the transaction (although that may be the default implementation). As you said, the runtime must implement a `Convert<Weight, Balance>` function, but it does not have to be a simple multiplier.

EDIT: I see this mistake existed before. Sorry :). But still worth fixing.